text/0012-CLI-commands.md

```diff
@@ -0,0 +1,1367 @@
+
+Once you close a connection to a peer, you or that peer can immediately open another 
+connection.
+
+You must have `qri connect` running in another terminal.
+
+
+```
+qri peers disconnect <peername, peer_id, or multiaddr>
+```
+
+### Examples
+
+```
+  # disconnect from a peer using a multiaddr
+  qri peers disconnect /ip4/192.168.0.194/tcp/4001/ipfs/QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn
+```
+
+### Options
+
+```
+  -h, --help   help for disconnect
+```
+
+
+
+<a id='qri_peers_info'></a>
+## qri peers info
+
+Get info on a Qri peer
+
+### Synopsis
+
+
+The peers info command returns a peer's profile information. The default
+format is yaml.
+
+Using the `--verbose` flag, you can also view a peer's network information.
+
+You must have `qri connect` running in another terminal.
+
+```
+qri peers info [flags] <peername>
+```
+
+### Examples
+
+```
+  # spin up a Qri node in a separate terminal
+  qri connect
+
+  # then 
+  # show info on a peer named "b5":
+  qri peers info b5
+
+  # show info in json:
+  qri peers info b5 --format json
+```
+
+### Options
+
+```
+      --format string   output format. formats: yaml, json (default "yaml")
+  -h, --help            help for info
+  -v, --verbose         show verbose profile info
+```
+
+
+
+<a id='qri_peers_list'></a>
+## qri peers list
+
+List known qri peers
+
+### Synopsis
+
+
+Lists the peers to which your Qri node is connected. 
+
+You must have `qri connect` running in another terminal.
+
+To find include peers that are not online, but to which your node has previously been 
+connected, use the `--cached` flag.
+
+```
+qri peers list [flags]
+```
+
+### Examples
+
+```
+  # spin up a Qri node
+  qri connect
+
+  # thenin a separate terminal, to list qri peers:
+  qri peers list
+
+  # to ensure you get a cached version of the list:
+  qri peers list --cached
+```
+
+### Options
+
+```
+  -c, --cached           show peers that aren't online, but previously seen
+  -h, --help             help for list
+  -l, --limit int        limit max number of peers to show (default 200)
+  -n, --network string   specify network to show peers from [ipfs]
+  -s, --offset int       number of peers to skip during listing
+```
+
+
+
+<a id='qri_registry'></a>
+## qri registry
+
+commands for working with a qri registry
+
+### Synopsis
+
+Registries are federated public records of datasets and peers.
+These records form a public facing central lookup for your datasets, so others
+can find them through search tools and via web links. You can use registry 
+commands to control how your datasets are published to registries, opting out
+on a dataset-by-dataset basis.
+
+By default qri is configured to publish to https://registry.qri.io,
+the main public collection of datasets & peers. "qri new" and "qri update"
+default to publishing to a registry as part of dataset creation unless run 
+with the "no-publish" flag. Published datasets that are under 20MBs will be
+backed up by the Qri registry.
+
```

Comment:
We've talked about backing up published datasets under a certain size before, and I just picked an arbitrary number.

Not married to leaving this in.

Reply:
Yeah the more I've thought about this, the more I think we need the concept of a "dataset preview" that is the head of a dataset, plus a sampling of the body (the hard part is getting "sampling" right). I'd like to get into a position where we can say the registry will _always_ accept the preview of a dataset, and if the registry likes you enough, it'll take the whole thing. For that we'd need _another_ RFC that outlines what "likes you" means, which is the concept of reputation.

So, we have a bunch of work to do. For now I think we should just say "a preview of the dataset", and leave it as a murky thing, because right now it's a murky thing.

Reply:
could also just remove it 🤷‍♂️

Reply:
I think maybe we remove it since we have https://github.com/qri-io/rfcs/issues/16 documenting as a record of dataset previews as a new feature